docs/release-source/release/stubs.md

```diff
@@ -242,6 +251,30 @@ Causes the stub to throw an exception of the provided type.
 Causes the stub to throw the provided exception object.
 
 
+#### `stub.rejects();`
+
+Causes the stub to return a Promise which rejects with an exception (`Error`).
+
+When constructing the Promise, sinon uses the `Promise.reject` method. You are
+responsible for providing a polyfill in environments which do not provide `Promise`.
+
+*Since `sinon@2.0.0`*
+
+
+#### `stub.rejects("TypeError");`
+
+Causes the stub to return a Promise which rejects with an exception of the provided type.
+
+*Since `sinon@2.0.0`*
+
+
+#### `stub.rejects(obj);`
```

Comment:
Similar to above, this could be any value. I propose that you rename  the argument to be `value`.